src/m365/spo/commands/roledefinition/roledefinition-add.ts

```diff
@@ -0,0 +1,134 @@
+import { Logger } from '../../../../cli';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import SpoCommand from '../../../base/SpoCommand';
+import { BasePermissions, PermissionKind } from '../../base-permissions';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  name: string;
+  description?: string;
```

Comment:
Let's add the `rights` option to the interface

Reply:
done